README.md

```diff
@@ -4,6 +4,8 @@
 
 [<img src="./Images/macos_badge.png" alt="download gordian wallet for macos" width="100"/>](./GordianWallet-macOS.dmg)<br/>
 
+<img src="https://github.com/Fonta1n3/GordianWallet-iOS/blob/master/Images/GordianWallet_logo.png" alt="Logo" width="300"/><br/>
```

Comment:
This is still not a local repo link:
```
<img src="https://github.com/Fonta1n3/GordianWallet-iOS/blob/master/Images/GordianWallet_logo.png" alt="Logo" width="300"/>
```

It needs to be either to `github.com/BlockchainCommons/GordianWallet-iOS`, or more ideally just use a local link to `./blob/master/Images/GordianWallet_logo.png` as that will work for all repos.

Reply:
@bcksenanayake — This PR has gotten too complex, especially since it has been added to by @Fonta1n3 — it was intended as a first easy PR for a new contributor, but has gotten out of hand. But hopefully a learning lesson for us all.

I suggest that you close this PR, and do a new, simpler PR. Start based on the root branch master, and create a PR that just has two items in a single commit. The image, and the change to README with either a LOCAL path to the image or the full path to the image that points to the root repo.

The other thing, as this is a security project, is to make sure that you have git signed your commit and that your gpg public keys are in GitHub. We don't require a git-signed CLA in `./CLA-signed/` for contributions to fix typos and minor documentation changes, but larger changes also require that you have signed and submitted a CLA first.

Thank you for your willingness to take on this, what should be, a small task, and hope it doesn't turn you off from contribution to open source or to our community.

-- Christopher Allen